python/ray/tune/tune.py

```diff
@@ -307,15 +309,16 @@ def run_experiments(experiments,
 
     trials = []
     for exp in experiments:
-        trials += run(exp,
-                      search_alg=search_alg,
-                      scheduler=scheduler,
-                      with_server=with_server,
-                      server_port=server_port,
-                      verbose=verbose,
-                      resume=resume,
-                      queue_trials=queue_trials,
-                      reuse_actors=reuse_actors,
-                      trial_executor=trial_executor,
-                      raise_on_failed_trial=raise_on_failed_trial)
+        trials += run(
+            exp,
+            search_alg=search_alg,
+            scheduler=scheduler,
+            with_server=with_server,
+            server_port=server_port,
+            verbose=verbose,
+            resume=resume,
+            queue_trials=queue_trials,
+            reuse_actors=reuse_actors,
+            trial_executor=trial_executor,
+            raise_on_failed_trial=raise_on_failed_trial)
```

Comment:
It's wired that after I running `scripts/format.sh` at my local on the latest commit, it seems that all your modifications in this commit are reversed and turn to my latest "formatting" commit.

I am using macOS Mojave with python 3.6.0, latest Ray.